docker-compose.yaml

```
@@ -0,0 +1,8 @@
+version: "3.8"
+services:
+  web:
+    build: .
+    ports:
+      - "8123:8123"
+    env_file:
+      - .env
```

```suggestion
    ports:
      - "$PORT:8123"
```
Same suggestion as the prod file, but could leave the port the same if you'd prefer